jstest/__main__.py

```diff
@@ -16,6 +16,7 @@
 
 import argparse
 import atexit
+import os
```

Comment:
You can remove this import. The `utils` module is already imported, and there is a `utils.abspath` function that you can use.